noxfile.py

```diff
@@ -110,9 +110,10 @@ def system(session):
 
     # Install all test dependencies, then install this package into the
     # virtualenv's dist-packages.
-    session.install("mock", "pytest", "psutil")
+    session.install("mock", "pytest")
 
     session.install("-e", "test_utils")
+    session.install("-e", "psutil")
```

Comment:
Psutil is an external test dependency, why has it been changed to an editable install? This looks like something that could fail.

**Edit:** Indeed, that's the cause of the failing system tests.

Reply:
@hongalex I think you were looking at the psutil issue?

Reply:
Hm, yeah I added psutil in [this PR](https://github.com/googleapis/python-pubsub/pull/55) but I didn't realize it was an editable install. Every time we try to merge in a gapic generated PR, it tries to remove psutil from the dependencies, so we have to manually add it in. 

@busunkim96, I tried looking for other places where psutil should be (re)added, but can't seem to find it. Any ideas?

Reply:
I asked Alex to add the extra dependency via synth.py so it doesn't get overridden in subsequent PRs. But the noxfile template incorrectly assumes that all extra dependencies are local. :) I'll open a PR in synthtool.

https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/noxfile.py.j2

Reply:
https://github.com/googleapis/synthtool/pull/469

Reply:
Thanks for the quick fix of the synthtool!

Reply:
I pushed a commit with the regeneration. PTAL and merge when you're ready. :)